app/models/project.rb

```diff
@@ -13,6 +13,7 @@ class Project < ActiveRecord::Base
 
   EXPERT_ROLES = [:expert, :owner]
 
+  belongs_to :default_tutorial, class_name: "Tutorial"
```

Comment:
Sure if you'd prefer it the other way I'll switch it around. It made more sense to me that the project had the default defined on it, but I can see the other way too.